elastic/integrations
# Add nats package

## What does this PR do?
This PR adds the NATS package.

## Checklist


## How to test this PR locally

1. Manually start Elastic stack and Nats service 
2. Enable NAts integration from Fleet UI
3. Verify data are populated and both dashboards work.

## Related issues

- Closes https://github.com/elastic/integrations/issues/359


## Screenshots
<img width="1281" alt="Screenshot 2020-11-25 at 11 23 09" src="https://user-images.githubusercontent.com/11754898/100224070-fe7f0f00-2f24-11eb-8e12-e24bf013e668.png">
<img width="843" alt="Screenshot 2020-11-25 at 12 22 57" src="https://user-images.githubusercontent.com/11754898/100224080-02ab2c80-2f25-11eb-8d44-15b68b861d27.png">
<img width="843" alt="Screenshot 2020-11-25 at 12 23 09" src="https://user-images.githubusercontent.com/11754898/100224086-0474f000-2f25-11eb-8e96-e5e587997804.png">
<img width="1233" alt="Screenshot 2020-11-25 at 12 24 15" src="https://user-images.githubusercontent.com/11754898/100224090-063eb380-2f25-11eb-8dea-04d4722307f5.png">
<img width="1654" alt="Screenshot 2020-11-25 at 13 45 32" src="https://user-images.githubusercontent.com/11754898/100224095-08a10d80-2f25-11eb-9b05-66f4730ea88d.png">
<img width="1654" alt="Screenshot 2020-11-25 at 13 47 47" src="https://user-images.githubusercontent.com/11754898/100224102-0c349480-2f25-11eb-8cfa-b99f92925d88.png">
